src/client/components/household-information/FinancialDisclosureSection.jsx

```diff
@@ -14,38 +14,15 @@ import { veteranUpdateField } from '../../actions';
 class FinancialDisclosureSection extends React.Component {
   render() {
     let content;
-    let understandsFinancialDisclosure;
-    let understandsFinancialDisclosureReview;
-
-    if (this.props.data.provideFinancialInfo.value === 'N') {
-      understandsFinancialDisclosure = (<div>
-        <ErrorableRadioButtons
-            label="I understand VA is not currently enrolling new applicants who decline to provide their financial information unless they have other qualifying eligibility factors."
-            name="understandsFinancialDisclosure"
-            options={yesNo}
-            value={this.props.data.understandsFinancialDisclosure}
-            onValueChange={(update) => {this.props.onStateChange('understandsFinancialDisclosure', update);}}/>
-      </div>);
-    }
-
-    if (this.props.data.provideFinancialInfo.value === 'N') {
-      understandsFinancialDisclosureReview = (<tr>
-        <td>I understand VA is not currently enrolling new applicants who decline to
-        provide their financial information unless they have other qualifying eligibility factors: </td>
-        <td>{`${this.props.data.understandsFinancialDisclosure.value === 'Y' ? 'Yes' : 'No'}`}</td>
-      </tr>);
-    }
 
     if (this.props.isSectionComplete && this.props.reviewSection) {
       content = (<table className="review usa-table-borderless">
         <tbody>
           <tr>
-            <td>I agree to provide my financial information so the VA can determine my eligibility
-             for VA healthcare and if I should be charged for copays and medication.:
-            </td>
-            <td>{`${this.props.data.provideFinancialInfo.value === 'Y' ? 'Yes' : 'No'}`}</td>
+            <td>I understand VA is not currently enrolling new applicants who decline to
```

Comment:
No, we will wait to update that language until we get the full list of changes from her, which I have not received yet.